airbyte-integrations/connectors/source-monday/source_monday/manifest.yaml

```diff
@@ -63,8 +63,10 @@ definitions:
       paginator:
         $ref: "#/definitions/default_paginator"
         pagination_strategy:
-          $ref: "#/definitions/default_paginator/pagination_strategy"
-          page_size: 1
+          class_name: "source_monday.item_pagination_strategy.ItemPaginationStrategy"
+          type: "CustomPaginationStrategy"
+          page_size: 100
+          start_from_page: 1
```

Comment:
This is not clear the page and subpage will start from 1. Any reason why you'd like to have control over this parameter?

Reply:
Both :-)
I will introduce another parameter for sub-page

Reply:
You are right. With a custom pagination strategy, this parameter makes little sense. I will embed this value into the strategy.